prow/cluster/build_serviceaccounts.yaml

```diff
@@ -9,4 +9,13 @@ metadata:
   name: k8s-gcr-audit-test-prod
   namespace: test-pods
 ---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  annotations:
+    # Used by Kops testing jobs
+    iam.gke.io/gcp-service-account: pr-kubekins@kubernetes-jenkins-pull.iam.gserviceaccount.com
+  name: k8s-kops-test
```

Comment:
Consider adding `serviceAccountName:` to your jobs now/next PR.

Next step will be to run the [bind script](/workload-identity) and then you'll be able to delete the `GOOGLE_APPLICATION_CREDENTIALS` stuff (env and related secret)